index.html

```diff
@@ -331,6 +142,41 @@ <h2 class="inter">
 				</li>
 			</ul>
 		</footer>
+		<div id="modal">
```

Comment:
Kindly update your mobile popup to match the figma design better🙏

Your design:
<img width="237" alt="image" src="https://github.com/hamidazim321/Portfolio-Project/assets/37116322/736ef946-a3f7-45ef-94c5-8a6ed0ad782b">

Figma design:
<img width="167" alt="image" src="https://github.com/hamidazim321/Portfolio-Project/assets/37116322/6a10234e-f8bc-4eae-9bd7-8f116b4d1d65">

A few things to note:
- Please make sure that all elements in your modal are using the correct font size, font weight and font family as seen in the figma design
- Please reduce the internal padding on the modal

Reply:
@Shisui6 Hey bro, should I change the Modal function to load the same Modal for each project we click? Because the Design does not look exactly the same as the Figma design when there are different lengths of text and image sizes in the modal for each Project. Should I change it to load the same Modal for each project card?

Reply:
@hamidazim321 Each button should load the content for a different card. You can give each project the same description to make it easier  to style, however the titles of the project should be different as well as the images. Try to use CSS to create style rules that accommodate for different lengths in project titles. As for images,they should all be the same size👍